copy/copy.go

```diff
@@ -604,11 +598,12 @@ func (c *copier) createProgressBar(pool *mpb.Progress, info types.BlobInfo, kind
 	}
 
 	bar := pool.AddBar(info.Size,
+		mpb.BarClearOnComplete(),
 		mpb.PrependDecorators(
-			decor.Name(prefix),
+			decor.OnComplete(decor.Name(prefix), fmt.Sprintf("%s %s", prefix, onComplete)),
```

Comment:
(With my past localization experience (which, I realize, has no relevance to what we are doing at all), I’m conceptually not thrilled about just appending a “completed” word to an existing sentence. Already the `"%s", kind` is pushing it.)

Also, it just seems conceptually cleaner to add the final status to the `AppendDecorators`: it used to show progress, and that is replaced by a “done” message. Removing the progress and moving “done” to the end of prefix seems more convoluted to me.

(AFAICS the progress bar just concatenates prefix and suffix, but I might be missing something.)

Reply:
I like the idea! Moving it to the `AppendDecorators` seems more idiomatic.

> [...] I’m conceptually not thrilled about just appending a “completed” word to an existing sentence [...]

Any suggestion? I couldn't find an alternative tbh.

Reply:
(I don’t really have a suggestion WRT sentence pasting; moving the state to `AppendDecorator`… conceptually works, but the string on the screen ends up being a single sentence anyway.

A full l10n-safe fix would be not do do splitting at all, and have localizable strings like
- Copying blob $digest
- Copying blob $digest done
- Copying blob $digest skipped: already exists
and two more for configs.

We are not doing l10n at all, so this would be an absolute overkill right now. Let’s not worry about it.)